packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -117,9 +116,6 @@ export default abstract class SqlIntegration
     sql: string,
     setExternalId?: ExternalIdCallback
   ): Promise<QueryResponse>;
-  async cancelQuery(externalId: string): Promise<void> {
-    logger.debug(`Cancel query: ${externalId} - not implemented`);
-  }
```

Comment:
Getting rid of the abstract definition so we can more easily check whether an integration has a `cancelQuery` defined or not